boxsdk/object/api_json_object.py

```diff
@@ -0,0 +1,12 @@
+# coding: utf-8
+
+from __future__ import unicode_literals
+from collections import Mapping
+
+from boxsdk.object.base_api_json_object import BaseAPIJSONObject
```

Comment:
With `absolute_import` turned on, this should be changed to `from .base_api_json_object import BaseAPIJSONObject`.